### ISSUE TITLE
Have beta blockers improve ranged accuracy and lung capacity.

### ISSUE THREAD

**Issue by zachary-kaelan** [CleverRaven/Cataclysm-DDA#74564]
### Is your feature request related to a problem? Please describe.

Beta blockers are on the [official list of prohibited substances](https://www.wada-ama.org/en/prohibited-list) for a number of sports, chiefly involving fine accuracy and underwater sports, due to the physiological benefits of lower heart rate for those sports.

### Solution you would like.

According to [this study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5407199/) on elite divers, the amount of time someone can hold their breath is increased by up to 20% with an average of 10%.

According to [this study](https://pubmed.ncbi.nlm.nih.gov/2875053/) on marksmen, accuracy while shooting is improved by 13.4%.

There is [also a study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8108660/) on archery, which shows no change in accuracy. We can choose to sweep this under the rug and give benefits to archery anyway, if it's easier to code or to avoid giving archery another reason to be inferior.

These all seem like very reasonable numbers from a game balance standpoint.

### Describe alternatives you have considered.

Doing nothing better with our free time.

**Comment by kevingranade** [comment]
Nice finds, though I think you might be reading into it a bit more than is warranted.
The study on pistol accuracy seens like it would only be applicable for pistols in particular, especially as the abstract mentions the effect seems to stem just from reduction in hand tremor.  Anything with a stock (riflrs, shotguns, smgs, even pistols with stocks or braces) is already working to cancel this effect out, so would see little to no improvement. We definitely would not apply it to archery. 
As for magnitude,  that 13% is "score improvement" on some particular shooting contest,  need to see the details to tease out an effect in minutes of arc.

**Comment by Venera3** [comment]
Apjea tolerance is measured on "elite apnea contestants" with a median apnea length of six minutes and an end-apnea SpO2 of 78%, using both an iv loading dose and a continuous infusion, *and* while not doing anything otherwise.
Needless to say, you're not that good, you're not hogging a perfusor around, and the main factor of your oxygen use while swimming *isn't* your resting heart rate.

For the marksmanship value it's looking a bit better, but even the abstract points out that the more skilled marksmen showed improvement, and it's using a 150mg oral dosage (way above the normal starter dose) on naive subjects - given that beta-blockers have a pretty hefty tolerance buildup that effect will only get smaller.

**Comment by kholat** [comment]
Good idea, beta blockers are banned in competitive shooting by the ISSF.

**Comment by zachary-kaelan** [comment]
I did some more research, and it seems that the improvements to performance are primary linked to a reduction in performance anxiety. 

[10-1108_PIJPSM-03-2013-0034.pdf](https://github.com/user-attachments/files/15958188/10-1108_PIJPSM-03-2013-0034.pdf)

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/30245333/adce07a2-d05f-45d6-9fba-c43471bcf0dd)

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/30245333/cf49b212-1e89-49f8-aca1-4771f8b9b63b)

That paper showed "change in anxiety" (performance anxiety) to be a very important factor, alongside state anxiety, average heart rate, maximal heart rate, and change in heart rate (the rest being irrelevant to beta blockers).
